<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plotter</class>
 <widget class="QDialog" name="Plotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	// background-color: ;	
}
</string>
  </property>
  <widget class="QCustomPlot" name="myGraph" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>631</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCustomPlot: {
	background-color: white;
	border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_min">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  color: rgb(208, 251, 225);	
font: 500 15px  &quot;Quicksand&quot;;</string>
   </property>
   <property name="text">
    <string>-100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_max">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  color: rgb(208, 251, 225);	
font: 500 15px  &quot;Quicksand&quot;;</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="y_min">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  color: rgb(208, 251, 225);	
font: 500 15px  &quot;Quicksand&quot;;</string>
   </property>
   <property name="text">
    <string>-100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="y_max">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>460</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  color: rgb(208, 251, 225);	
font: 500 15px  &quot;Quicksand&quot;;</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  color: rgb(208, 251, 225);	
font: 500 15px  &quot;Quicksand&quot;;</string>
   </property>
   <property name="text">
    <string>Domain</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  color: rgb(208, 251, 225);	
font: 500 15px  &quot;Quicksand&quot;;</string>
   </property>
   <property name="text">
    <string>Codomain</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
